packages/components/src/components/PackageHeader.tsx

```diff
@@ -149,7 +142,7 @@ const TagListing: React.FC<TagsProps> = (props) => {
   // in a modal window.
   return (
     <Box alignSelf="reverse" mt={renderFullWidth ? "0" : "20px"}>
-      {visible.map((t) => tag(t))}
+      {visible.map(tag)}
```

Comment:
I'm guessing this and similar changes elsewhere are for simplifying code while you're at it, which is cool.